### ISSUE TITLE
timestamp-millis bug introduced in 0.14. Works in 0.13

### ISSUE THREAD

**Issue by kurtostfeld** [tebeka/fastavro#81]
The following simple program worked perfectly in fastavro 0.13 and now fails on fastavro 0.14. I presume this is a bug?

```python
from io import BytesIO

import datetime
import fastavro

avro_schema = {
  "type": "record",
  "name": "simple_record",
  "namespace": "org.test",
  "fields": [{
    "name": "name",
    "type": "string"
  }, {
    "name": "timestamp",
    "type": {
      "type": "long",
      "logicalType": "timestamp-millis"
    }
  }]
}

epoch = datetime.datetime.utcfromtimestamp(0)


def datetime_to_epoch_millis(dt):
    return long((dt - epoch).total_seconds() * 1.0e3)


def generate_avro():
    avro_dict = {
        "name": "abc",
        "timestamp": datetime_to_epoch_millis(datetime.datetime(2015, 6, 7, 15, 30, 0))
    }

    with BytesIO() as bytes_io:
        fastavro.schemaless_writer(bytes_io, avro_schema, avro_dict)
        return bytes_io.getvalue()


if __name__ == "__main__":
    print("generate_avro test...")
    avro_binary = generate_avro()
    print("done :)")
    print("length={}".format(len(avro_binary)))
```

**Comment by tebeka** [comment]
@kkirsanov ?

**Comment by tebeka** [comment]
Thanks for reporting, will check.

**Comment by kkirsanov** [comment]
Hmm.
It`s not a bug but "Not obvious behavior" Avro`s serializer expects datetime object not long

I think the best solution is to leave the integers unchanged

**Comment by kurtostfeld** [comment]
Version 0.13 would throw exception when writing a datetime value rather than a long.

**Comment by kkirsanov** [comment]
Thanks for testing.

Fixed in 0.14.2
